src/ReqQueryParams/ReqQueryParams.js

```diff
@@ -22,4 +24,39 @@ export class ReqQueryParams {
       return defaultValue
     }
   }
+
+  getInteger(name, defaultValue) {
+    let param, value
+    try {
+      param = this.get(name)
```

Comment:
True, same for getBoolean

Reply:
Keep in mind that the `defaultValue` in `getBoolean` could be `true` or `false` instead of a string representation of them and the assignment of `value` will fail. That is why I only re-think `getInteger`